cmd/wio/toolchain/npm/resolve/resolve.go

```diff
@@ -42,13 +92,36 @@ func (i *Info) ResolveRemote(config types.IConfig) error {
 
     root := &Node{name: config.Name(), ver: config.Version()}
     if root.resolve = semver.Parse(root.ver); root.resolve == nil {
-        return errors.Stringf("project has invalid version %s", root.ver)
+        return errors.Stringf("project has an invalid version %s", root.ver)
     }
+
+    // remotes from main config file
     deps := config.Dependencies()
     for name, ver := range deps {
+        // only worry about remotes
+        if strings.Contains(ver, io.Vendor) {
```

Comment:
versions will never contain "vendor" since it's an invalid version string, shouldn't you use the Vendor flag?